### ISSUE TITLE
Still getting the "No subscriptions" error

### ISSUE THREAD

**Issue by BobWs** [iganeshk/standardnotes-extensions#36]
Closing this as PR https://github.com/iganeshk/standardnotes-extensions/pull/35 will be merged shortly. If anyone is still having issues feel free to open a new issue.

_Originally posted by @xthursdayx in https://github.com/iganeshk/standardnotes-extensions/issues/33#issuecomment-1003206545_

**Comment by BobWs** [comment]
Hi,
I just updated my Standardnotes Stack and now I'm getting the error again on standardnotes-web version.
Before updating standardnotes I first updated the extensions after reading about the problems here, but now I see the errors again.

Is there a way to fix this? Did I missed something?

**Comment by gyaneesh** [comment]
Did you pull the latest changes of this repository and then run it? (git fetch && git pull)

**Comment by xthursdayx** [comment]
You will also need to make sure to deactivate and uninstall  the editors or themes that are causing the errors and then reinstall  them from your self-hosted URL, after running the updated container.

**Comment by BobWs** [comment]
> Did you pull the latest changes of this repository and then run it? (git fetch && git pull)

Yes I did and it was working fine until I updated standardnotes containers

**Comment by BobWs** [comment]
> You will also need to make sure to deactivate and uninstall the editors or themes that are causing the errors and then reinstall them from your self-hosted URL, after running the updated container.

Okay I will give it a try and see what happens...

**Comment by BobWs** [comment]
Okay So now I’m getting a 404 error page in standardnotes-web, and I’m not able to install any extensions.

I have installed the Docker version of your repo and it seems to be working fine, no errors in the logs. And when I browse via a reverse proxy subdomain to my extensions I get a list of all the extensions which are installed/ pulled in the public folder. So that indicates that it is working.

But I don’t get why I’m getting a 404 page! And no able to install the extensions.

**Comment by BobWs** [comment]
So Kind of solved the Standardnotes-Web problem! I revered back to an older version and re-entered the extensions, it was a pain in the ** because I had to re-link all my notes (500+) with the editors. But okay it works again, the 404 pages are gone.

The only problem that remains is that I cannot install the extension in standardnotes-web, I can only install them via the Desktop app. So that is kind a weird.
